packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -16,6 +17,60 @@ class UsaBanner extends React.PureComponent {
   }
 
   render() {
+    const tld = `.gov`;
```

Comment:
Translation content should be inside a .json file similar to in `hcgoc-design-system`. See this as an example: https://github.com/CMSgov/hcgov-design-system/tree/master/src/locale

Reply:
I would also consider using a translation framework like [react-i18next](https://react.i18next.com/guides/quick-start). While this is the only component that needs internationalization in the DS, a framework would standardize how we do translations for the future.

Reply:
I'm Wondering if we should hold off on using a translation framework until we have a more established need for it beyond this usa banner?

Reply:
Add translation framework react-i18next to this branch [WNMGDS-698/add-USA-banner-spanish-prop-i18n](https://github.com/CMSgov/design-system/tree/WNMGDS-698/add-USA-banner-spanish-prop-i18n). While this works, it make be better to not implement this yet as the latest version of react-i18next uses hooks/context and we're not looking to update yet. For the test branch, I'm used the older version of react-i18next.

Reply:
@bernardwang @line47 - we only have Usa Banner that needs translation, I prefer to keep it simple for now by using the .json files. We can add the framework when we have more components that requires translation or when we've upgraded to later React version.

Reply:
I'd prefer to use `react-i18next` as it contains a feature to prevent splitting translations (as referenced [here](https://github.com/CMSgov/design-system/pull/888#discussion_r537647678)). It's also a far more standardized approach to translation that I feel we should eventually adopt anyway.

 However, in the interest of finishing this PR, @nichia we can go with this solution. In the meantime, we should ask engineers what they prefer from the UsaBanner and do some more research on how internationalized components could work in an application.